pages/api/auth/[...nextauth].js

```diff
@@ -70,4 +70,7 @@ export default NextAuth({
       logger.debug(metadata)
     },
   },
+  pages: {
+    signIn: '/signedout',
+  },
```

Comment:
What does this do?

Reply:
This is what makes next-auth redirect to the custom signIn page.